content/desktop/install/windows-install.md

```diff
@@ -107,15 +107,41 @@ Running Docker Desktop inside a VMware ESXi or Azure VM is supported for Docker
 It requires enabling nested virtualization on the hypervisor first.
 For more information, see [Running Docker Desktop in a VM or VDI environment](../vm-vdi.md).
 
-{{< accordion title="About Windows containers" >}}
+{{< accordion title="How do I switch between Windows and Linux containers?" >}}
 
-Looking for information on using Windows containers?
+From the Docker Desktop menu, you can toggle which daemon (Linux or Windows)
+the Docker CLI talks to. Select **Switch to Windows containers** to use Windows
+containers, or select **Switch to Linux containers** to use Linux containers
+(the default).
 
-* [Switch between Windows and Linux containers](../faqs/windowsfaqs.md#how-do-i-switch-between-windows-and-linux-containers)
-  describes how you can toggle between Linux and Windows containers in Docker Desktop and points you to the tutorial mentioned below.
-- [Getting Started with Windows Containers](https://learn.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment?tabs=dockerce)
-- Docker Container Platform for Windows [articles and blog
-  posts](https://www.docker.com/microsoft/) on the Docker website.
+For more information on Windows containers, refer to the following documentation:
+
+- Microsoft documentation on [Windows containers](https://docs.microsoft.com/en-us/virtualization/windowscontainers/about/index).
+
+- [Build and Run Your First Windows Server Container (Blog Post)](https://blog.docker.com/2016/09/build-your-first-docker-windows-server-container/)
```

Comment:
This link redirected to the main blog page for me. But when I searched for this blog title I found this link maybe will work better: https://www.docker.com/blog/build-your-first-docker-windows-server-container/